lib/ruby_lsp/requests/hover.rb

```diff
@@ -131,6 +147,37 @@ def generate_hover(name, location)
           contents: markdown_from_index_entries(name, entries),
         )
       end
+
+      sig { params(node: Prism::CallNode).void }
+      def generate_gem_hover(node)
+        first_argument = node.arguments&.arguments&.first
+        return unless first_argument.is_a?(Prism::StringNode)
+
+        spec = resolve_gem_spec(first_argument)
```

Comment:
And then get rid of `resolve_gem_spec`.
```suggestion
        spec = Gem::Specification.find_by_name(first_argument.content)
```